elasticapm/metrics/sets/cpu.py

```diff
@@ -0,0 +1,7 @@
+import os
+import platform
+
+if platform.system() == "Linux" and "ELASTIC_APM_FORCE_PSUTIL_METRICS" not in os.environ:
```

Comment:
why not always use `psutil` if available, even on linux?

Reply:
as it is right now, psutil returns somewhat weird numbers (e.g. process CPU utilization is always 0.0). We might even have to yank psutils out completely :(

Reply:
with 7ab887d, psutil should now also be operational. I still think we should default to our own `/proc/` parsing on Linux, as psutil is pretty humongous

Reply:
I can live with that, though I do like that psutil handles older formats of `/proc/stat`, etc.  Probably premature but it would be neat if we could use psutil if available if we're sure that the homegrown routines won't work (like on older kernels)